### PR TITLE
update comment

### PR DESCRIPTION
**Pull request by iuhilnehc-ynos**

### CODE REVIEW

File: rosidl_typesupport_introspection_c/resource/srv__type_support.c.em [hunk 1]
```diff
@@ -61,7 +61,7 @@ function_prefix = '__'.join(include_parts) + '__rosidl_typesupport_introspection
 static rosidl_typesupport_introspection_c__ServiceMembers @(function_prefix)__@(service.namespaced_type.name)_service_members = {
   "@('__'.join([package_name] + list(interface_path.parents[0].parts)))",  // service namespace
   "@(service.namespaced_type.name)",  // service name
-  // these two fields are initialized below on the first access
+  // these three fields are initialized below on the first access
```
Comment (iuhilnehc-ynos) [inline rosidl_typesupport_introspection_c/resource/srv__type_support.c.em:64]: Maybe use 'the following fields' instead.

Comment (clalancette) [reply rosidl_typesupport_introspection_c/resource/srv__type_support.c.em:64]: Yeah, agreed.  I think this should be:

```suggestion
  // the following fields are initialized below on first access
```

**Review by clalancette — CHANGES_REQUESTED** [review]

**Review by emersonknapp — APPROVED** [review]
Good change - any number written in a comment inevitably becomes incorrect

**Review by clalancette — APPROVED** [review]

**Comment by clalancette** [comment]
CI:
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/19184/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/13753/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/19936/)

**Comment by clalancette** [comment]
Going ahead and merging this one (the timers_manager test failure is a known Windows flake).

**Pull request merged**